enviroment-files/q2-pinocchio-qiime2-amplicon-2025.10.yml

```diff
@@ -0,0 +1,12 @@
+name: q2-pinocchio-amplicon
+channels:
+  - https://packages.qiime2.org/qiime2/2025.10/amplicon/released  
+  - conda-forge
+  - bioconda
+dependencies:
+  - qiime2-amplicon
+  - minimap2
+  - q2-feature-classifier=2025.10
```

Comment:
Hey @ZuzanaSebb, these dependencies should be reflected in the conda recipe - there you still have nanoplot, samtools and others. You should either remove unnecessary stuff from there or add required packages here (or a combination of both).